packages/nimble-components/src/wafer-map/tests/worker-renderer.spec.ts

```diff
@@ -0,0 +1,42 @@
+import type { Remote } from 'comlink';
+import { createMatrixRenderer } from '../modules/worker-renderer';
+import type { MatrixRenderer } from '../../../build/generate-workers/dist/esm/source/matrix-renderer';
+
+describe('MatrixRenderer worker:', () => {
+    let matrixRenderer: Remote<MatrixRenderer> | undefined;
```

Comment:
```suggestion
    let matrixRenderer: Remote<MatrixRenderer>;
```

Reply:
removed it
![image](https://github.com/ni/nimble/assets/110180309/b7cc034a-202c-4457-9d81-4b57189174bc)